sample_tracking/app/edit/page.tsx

```diff
@@ -138,54 +141,31 @@ export default function Edit() {
     }
 
     return (
-        <div className="add-sample-page-wrapper">
-            <p className="title">Edit sample {formData.sample_name}</p>
-
-
-            <div className="edit-tabs-wrapper">
-                <div className="edit-tab-group">
-                    <div onClick={() => setCurrentTab(1)} className={currentTab === 1 ? "edit-tab-wrapper edit-current-tab" : "edit-tab-wrapper"}>
-                        <div className="edit-slate-layer">
-                            <div className="edit-tab-content">
-                                <div className='edit-tab-text'>
-                                    Basic info
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <div onClick={() => setCurrentTab(2)} className={currentTab === 2 ? "edit-tab-wrapper edit-current-tab" : "edit-tab-wrapper"}>
-                        <div className="edit-slate-layer">
-                            <div className="edit-tab-content">
-                                <div className='edit-tab-text'>
-                                    Sample measurements
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <div onClick={() => setCurrentTab(3)} className={currentTab === 3 ? "edit-tab-wrapper edit-current-tab" : "edit-tab-wrapper"}>
-                        <div className="edit-slate-layer">
-                            <div className="edit-tab-content">
-                                <div className='edit-tab-text'>
-                                    Review
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+        <div>
+            <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0&display=optional" />
```

Comment:
is it legit to be accessing material design via this url?  Is it stable?

Reply:
i would have though that this would be inherited already from the css.

Reply:
Will address this in a followup pr- this line is in a few places in the code.